skrub/tests/test_table_vectorizer.py

```diff
@@ -714,6 +714,16 @@ def test_bad_specific_cols():
         TableVectorizer(specific_transformers=[(None, [0])]).fit(None)
 
 
+def test_sk_visual_block_():
+    X = _get_clean_dataframe()
+    vectorizer = TableVectorizer()
```

Comment:
here we could check the repr html of the unfitted estimator to check it works and cover the missing line that codecov is complaining about

```python
unfitted_repr = vectorizer._repr_html_()
assert 'TableVectorizer' in unfitted_repr
```